src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -161,4 +165,11 @@ class EthService(blockchain: Blockchain, blockGenerator: BlockGenerator, appStat
       highestBlock = appStateStorage.getEstimatedHighestBlock())))
   }
 
+  def sendRawTransaction(req: SendRawTransactionRequest): ServiceResponse[SendRawTransactionResponse] = {
+    import io.iohk.ethereum.network.p2p.messages.CommonMessages.SignedTransactions._
+    val signedTransaction = rlp.decode[SignedTransaction](req.data.toArray[Byte])
```

Comment:
A kind reminder of the now [hidden comment](https://github.com/input-output-hk/etc-client/pull/184#discussion_r117232357)